### DISCUSSION TITLE
funny thing

### DISCUSSION THREAD

**Discussion by xoidex** [3kh0/ext-remover#789] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
hi
i'm no expert but i'm pretty sure this link i found (devtools://devtools/bundled/devtools_app.html) could be of some use.
idk if you can find some uses just type em down there
tyvm

more:
devtools://devtools/bundled/devtools_app.html?uiDevTools=true&ws=
devtools://devtools/bundled/node_app.html
devtools://devtools/bundled/worker_app.html
devtools://devtools/bundled/inspector.html
devtools://devtools/bundled/js_app.html

**Comment by AshtonDavies** [comment, marked as answer]
Useless

**Reply by xoidex** [reply]
okay :P

**Reply by AshtonDavies** [reply]
Although I can't remember if the chaos exploit used that at all, it might have. https://chrose.netlify.app/detail/Chaos
